src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -3,10 +3,16 @@
 import java.io.IOException;
 
 import com.jme3.app.SimpleApplication;
+import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Vector3f;
 import com.jme3.network.Network;
 import com.jme3.network.Server;
 import com.jme3.network.serializing.Serializer;
 
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Spatial;
+import com.jme3.scene.shape.Box;
 import nl.tudelft.pixelperfect.client.ConnectListener;
 import nl.tudelft.pixelperfect.client.HelloMessage;
 import nl.tudelft.pixelperfect.client.ServerListener;
```

Comment:
Swapped them, but Im not getting warnings about this specific thing, are you also using the rules of checkstyle.xml located in the project? David also seems to have other checkstyle warnings than I do in another PR so just wondering